### PR TITLE
usage attribution logs spans

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Usage Attribution is now supported for ingested/indexed logs and spans

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/billing/usage_attribution.md
Comment (hestonhoffman) [inline content/en/account_management/billing/usage_attribution.md:59]:
```suggestion
- Data is shown by all tag keys selected. For example, by app and service.
```

File: content/en/account_management/billing/usage_attribution.md
Comment (hestonhoffman) [inline content/en/account_management/billing/usage_attribution.md:85]:
```suggestion
[2]: https://docs.datadoghq.com/api/v1/usage-metering/#get-hourly-usage-attribution
```
Moving this link back. Unfortunately, our pre-commit hook currently breaks these links by moving them down in the document. I have a PR up to fix it.

File: content/en/account_management/billing/usage_attribution.md
Comment (hestonhoffman) [inline content/en/account_management/billing/usage_attribution.md:129]:
```suggestion
```

**Review by hestonhoffman — APPROVED** [review]
A couple of small changes and then this is GTG

**Comment by dussault-antoine** [comment]
Can we hold off before merging this one ? It's actually a feature that is in beta
I'll add the label Do Not Merge
cc

**Pull request closed without merging**
